client/src/pages/Login/Login.tsx

```diff
@@ -1,30 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Input } from '@/components/Shared/Input';
 import Button from '@/components/Shared/Button';
 import * as S from './styles';
-import { Link } from '@/lib/Router';
+import { Link, Redirect, useHistory } from '@/lib/Router';
 import { githubLogin } from '@/lib/api/login/githubLogin';
+import { normalLogin } from '@/lib/api/login/normalLogin';
+import {
+  validateEmail,
+  validateInput,
+} from '@/utils/constant/validate/validation';
 
 const Login = () => {
   const GithubLogin = async () => {
     const oAuthURL = await githubLogin();
     window.location.href = oAuthURL;
   };
 
-  // TODO : 만약 로그인이 되어있는 유저면 마이페이지로 이동시키기?
-  // if (data) return;
+  const [error, setError] = useState({
+    email: false,
+    password: false,
+  });
+  const { historyPush } = useHistory();
+
+  const onSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    const target = e.target as typeof e.target & {
+      email: { value: string };
+      password: { value: string };
+    };
+    const email = target.email.value;
+    const password = target.password.value;
+    const emailValidation = validateEmail(email);
+    const passwordValidation = validateInput(password);
+    setError({ email: !emailValidation, password: !passwordValidation });
+    if (emailValidation && passwordValidation) {
+      try {
+        await normalLogin({ user_id: email, password });
+        historyPush('/');
+      } catch (err) {
+        window.alert(err.response.data.message);
```

Comment:
요고 토스티파이로 대체 가능합니다 이제


```suggestion
        notify('error', err.response.data.message);
```

Reply:
넵 다음 커밋에 적용해서 pr올리겠습니다!